pkg/sql/show_create.go

```diff
@@ -73,6 +73,9 @@ func ShowCreateTable(
 
 	f := tree.NewFmtCtx(tree.FmtSimple)
 	f.WriteString("CREATE ")
+	if desc.Unlogged {
+		f.WriteString("UNLOGGED ")
+	}
```

Comment:
I agree with @rohany. If the `UNLOGGED` keyword does nothing, I think it's misleading to label the table as `UNLOGGED` in `SHOW CREATE`. If you remove this, I think you can avoid storing anything new in a descriptor, no?

Reply:
thought i removed this on cleanup!